src/VisualStudio/Core/Def/ServicesVSResources.resx

```diff
@@ -1478,4 +1478,7 @@ I agree to all of the foregoing:</value>
   <data name="Parameter_information" xml:space="preserve">
     <value>Parameter information</value>
   </data>
+  <data name="Infer_from_context" xml:space="preserve">
+    <value>Infer from context</value>
+  </data>
```

Comment:
can you show a picture of the UI?  i'm tyring to figure out where the different UI strings show up.

Reply:
Screenshots added to the original post in the PR.